packages/conveyer/src/Conveyer.ts

```diff
@@ -516,7 +516,7 @@ class Conveyer extends Component<ConveyerEvents> {
       this.trigger("leaveStart");
     }
     // enter end
-    if (pos >= scrollSize - size && this.isReachEnd !== true) {
+    if ((pos >= scrollSize - size || scrollSize - size - pos < 1) && this.isReachEnd !== true) {
```

Comment:
If there is only one condition "scrollSize - size - pos < 1", the condition seems to be the same.